src/load/setup.py

```diff
@@ -26,7 +26,7 @@
     'License :: OSI Approved :: MIT License',
 ]
 
-DEPENDENCIES = []
+DEPENDENCIES = ["azure-core", "pyyaml", "azure-mgmt-network"]
```

Comment:
azure-mgmt-network is removed from azure-cli. Which function do you rely on?

Reply:
I think `azure-core` and `pyyaml` are added in `azure-cli`

Reply:
We are using azure-mgmt-network for completers of subnets.

Reply:
Please drop the dependance on azure-mgmt-network.

Reply:
Removed dependency on azure-mgmt-network